openmc/material.py

```diff
@@ -82,6 +86,8 @@ def __init__(self, material_id=None, name='', temperature=None):
         self._density = None
         self._density_units = ''
         self._depletable = False
+        self._volume = None
+        self._atoms = {}
```

Comment:
Yeah, I kind of debated this one. I'm not 100% sure I want an atoms property, so for now I'm holding off on exposing it. As you point out, it's only used internally right now.